internal/engine/analytics_reporter.go

```diff
@@ -30,6 +43,13 @@ func (ar *AnalyticsReporter) OnChange(ctx context.Context, st store.RStore) {
 	// wait until state has been kinda initialized
 	if !state.TiltStartTime.IsZero() && state.LastTiltfileError() == nil {
 		ar.started = true
+
+		opt, err := analytics.OptStatus()
```

Comment:
sooo this isn't great because it's another place to keep the chosen opt consistent (i.e. when user chooses one way or another we need to remember to update their Opt here as well), but seemed better than reading the `choice.txt` file on every OnChange...?

Reply:
It's fine to leave as-is in this PR since it won't matter until we get to the code that actually modifies the value. I kinda suspect we'll want to pull it into `EngineState`.